capstone/src/main/java/com/google/univiz/CollegeScoreCardData/CollegeScoreCardInterface.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.collegescorecarddata;
+
+import java.util.List;
+
+interface CollegeScoreCardInterface {
```

Comment:
You'll probably want the interface(s) themselves to be public so you can access them outside of whatever package they end up in.

Reply:
done!